### PR TITLE
Update announcement

### PR DESCRIPTION
**Pull request by m-sugawara**
## Overview
- add new delegate for customizing container view.
- make text alignment of Announcement be center.
- add Example for above changes.

## Screenshot
Simulator Screen Shot - iPhone Xʀ - 2019-09-02 at 19 45 15

### CODE REVIEW

**Comment by m-sugawara** [comment]
@quipper/aya-native-ios-devs 
Please review if you have time 🙏

File: Sources/Views/Cells/TextMessageCell.swift [hunk 1]
```diff
@@ -77,7 +77,10 @@ open class TextMessageCell: MessageContentCell {
                 messageLabel.setAttributes(attributes, detector: detector)
             }
             switch message.kind {
-            case .text(let text), .emoji(let text), .announcement(let text):
+            case .announcement(let text):
+                messageLabel.textAlignment = .center
+                fallthrough
```
Comment (manicmaniac) [inline Sources/Views/Cells/TextMessageCell.swift:82]: I think this is proper use of `fallthrough` keyword :+1:

File: Sources/Protocols/MessagesDisplayDelegate.swift [hunk 2]
```diff
@@ -96,6 +96,15 @@ public protocol MessagesDisplayDelegate: AnyObject {
     ///   The default image configured by this method is `?`.
     func configureAccessoryView(_ accessoryView: UIView, for message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView)
 
+    /// Used to configure the `ContainerView` of a `MessageContentCell` class.
+    ///
+    /// - Parameters:
+    ///   - containerView: The `MessageContainerView` of the cell.
+    ///   - message: The `MessageType` that will be displayed by this cell.
+    ///   - indexPath: The `IndexPath` of the cell.
+    ///   - messagesCollectionView: The `MessagesCollectionView` in which this cell will be displayed.
```
Comment (manicmaniac) [inline Sources/Protocols/MessagesDisplayDelegate.swift:105]: Neat 💯

**Review by manicmaniac — COMMENTED** [review]
LGTM :+1:

**Pull request merged**